src/main/kotlin/org/malachite/estella/services/TaskStageService.kt

```diff
@@ -76,6 +76,12 @@ class TaskStageService(
         return taskStage.tasksResult.map { it.task.toTaskDto() }
     }
 
+    fun getTasksByInterview(interviewId: String): List<TaskDto> {
+        val taskStage = interviewService.getInterview(UUID.fromString(interviewId)).applicationStage.tasksStage!!
+        return taskStage.tasksResult.map { it.task.toTaskDto() }
```

Comment:
Dlaczego nie zrobiłeś tego w jednym ładnym funkcyjnym łańcuszku? Plus nie każde interview musi być powiązane z taskStage - at least I think so, więc zrobienie !! na takim interview walnie 500, a lepiej tego unikać. Proponuję zrobić tak, że złożysz to do jednego łańcucha, od pewnego momentu będziesz robił z pytajnikami, a na końcu ?: z pustą listą.

Wiem, że takiej sytuacji być nie powinno, ale tak samo nie powinno być 6-krotnego dubla stage'y  XDDD

Reply:
don